Wikimate.php

```diff
@@ -1097,7 +1097,7 @@ public function getInfo($refresh = false, $history = null)
 
 			$this->edittoken = $page['edittoken'];
 
-			if (!isset($page['missing'])) {
+			if (!isset($page['missing']) && isset($page['imageinfo'])) {
```

Comment:
Do you think a comment would be warranted above this line? I'm not sure the behavior you described in #84 is intuitive: 

> when a file existed but was subsequently deleted or moved to another path, that the MW API does not return the `missing` property for the requested path anymore. I guess it only does that when the path never existed in the first place.